php/test/Slice/macros/Test.ice

```diff
@@ -20,7 +20,7 @@
 
 module Test
 {
-    class Default
```

Comment:
`default` is a PHP keyword, and class names are case-insensitive. So this had to be updated.
These Slice tests are never run cross, so instead of using `php:identifier`, I just changed the name to `YesDefault`.
To mirror the other class named `NoDefault` right under it.